content/c-sharp/concepts/strings/terms/split/split.md

```diff
@@ -0,0 +1,136 @@
+---
+Title: '.Split()'
+Description: 'Divides a string into an array of substrings based on specified delimeter(s).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Libraries'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Split()`** method breaks a string into substrings based on specified delimiter(s), which can be a character, an array of characters, or a string. It returns an array of strings containing the split substrings.
+
+## Syntax
+
+The various overloads of the `.Split()` method in C# are as follows:
+
+```pseudo
+Split(String[], Int32, StringSplitOptions)
+```
+
+Splits a string into a maximum number of substrings using the strings in an array as delimiters.
+
+```pseudo
+Split(Char[], Int32, StringSplitOptions)
+```
+
+Splits a string into a maximum number of substrings using the characters in an array as delimiters.
+
+```pseudo
+Split(String[], StringSplitOptions)
+```
+
+Splits a string into substrings based on the strings in an array.
+
+```pseudo
+Split(Char[])
+```
+
+Splits a string into substrings based on the characters in an array.
+
+```pseudo
+Split(Char[], StringSplitOptions)
+```
+
+Splits a string into substrings based on the characters in an array.
+
+```pseudo
+Split(Char[], Int32)
+```
+
+Splits a string into a maximum number of substrings based on the characters in an array. 
+
+- `StringSplitOptions`: Represents an enumeration that defines whether to include empty substrings `(StringSplitOptions.None)` or exclude them `(StringSplitOptions.RemoveEmptyEntries)`.
+- `int count`: Represents the maximum number of substrings to return.
```

Comment:
There is no usage of `int count` in above pseudo code examples, this can be confusing.

We should write the pseudo code in following way so that the datatype and the values can be more clear - 

```
Split(string[] separator, int count, StringSplitOptions options)
```